src/components/molecules/SpaceInfoText/SpaceInfoText.module.scss

```diff
@@ -11,13 +13,19 @@
   h1 {
     font-size: font.$size--xl;
     font-weight: font.$weight--md;
+    margin-bottom: 0;
   }
 }
 
 .moreLessSpan {
-  color: #707070;
+  color: theme.$c-default-text;
+  opacity: opacity.$md;
```

Comment:
I am not sure this is the same as
>  "More" button should be 50% of the text color (whether dark mode or light mode), not grey

this might just mean the saturation is halved, or the luminescence, not that the whole text is half transparent

Reply:
confirmed: https://doesitsparkle.slack.com/archives/C031U4S7D08/p1649759272152029